classes/Provider/TalkHelperProvider.php

```diff
@@ -22,10 +22,48 @@
     public function register(Container $app)
     {
         $app[TalkHelper::class] = function ($app) {
+            $categories = $app->config('talk.categories');
+
+            if ($categories === null) {
```

Comment:
What about something like:

```php
$categories = $app->config('talk.categories') ?: [
  'api'                => 'APIs (REST, SOAP, etc.)',
  'continuousdelivery' => 'Continuous Delivery',
  // etc.
];
```

Reply:
Could do, but it's just not very readable, so personally I avoid ternary statements like that.